test/NuGet.Core.Tests/NuGet.Packaging.Test/SigningTests/CertificateUtilityTests.cs

```diff
@@ -162,10 +166,12 @@ public void HasExtendedKeyUsage_WithEku_ReturnsTrueForEku()
             using (var certificate = SigningTestUtility.GenerateCertificate("test",
                 generator =>
                 {
-                    generator.AddExtension(
-                        X509Extensions.ExtendedKeyUsage.Id,
-                        critical: false,
-                        extensionValue: new ExtendedKeyUsage(KeyPurposeID.IdKPCodeSigning));
+                    var usages = new OidCollection { TestCertificateGenerator.IdKPCodeSigning };
+
+                    generator.Extensions.Add(
+                        new X509EnhancedKeyUsageExtension(
+                              usages,
+                              critical: true));
```

Comment:
With the exception of the test [`IsValidForPurposeFast_WithAnyExtendedKeyUsage_ReturnsFalseForCodeSigningEku`](https://github.com/NuGet/NuGet.Client/pull/2685/files#diff-1267bcc195ace161677a111cfd7b09cbR231) below, why did you change `critical` from `false` to `true` for these tests?  I don't think it's a big deal either way --- just curious.

Reply:
This wasn't on purpose. I will revert it to minimize the changes.